maestro-domain/src/test/java/bio/overture/maestro/domain/api/DefaultIndexerTest.java

```diff
@@ -496,50 +470,27 @@ void shouldIndexSingleStudy() {
         val studyId = "PEME-CA";
         val repoCode = "TEST-REPO";
         val filesRepository = getStubFilesRepository();
-        val a1 = getStudyAnalyses(studyId);
```

Comment:
same here, deleteing the  test logic to make it pass is not a good thing 
what am I missing ?

Reply:
This test used to test `indexStudyToFileCentric` which cannot be mocked because in DeafultIndexerTest, DefaultIndexer is not a mock, so buildFileCentricDocuments cannot be mocked, testing was failing. I think in this scenario, testing the entry method `indexStudy` makes more sense.

Reply:
as I mentioned before `indexStudyToFileCentric` should be private method we should be calling `indexStudy` only

Reply:
you can disable the analysis centric index in the test app.yaml